### DISCUSSION TITLE
Discord server available

### DISCUSSION THREAD

**Discussion by afpineda** [afpineda/OpenSourceSimWheelESP32#21] [Announcements]
A **Discord server** is available for the user community:

[https://discord.com/invite/sKQ4zK47DS](https://discord.com/invite/sKQ4zK47DS)

Please, report any server misconfiguration.

**Comment by willitheracer** [comment]
Hi, I have a few questions and would like to join the Discord server but the link is outdated. Could you update it?

**Reply by afpineda** [reply]
Yes. The link does not expire, but I will renew it.
You will have to wait since I'm not at home right now.
I'll answer your questions in GitHub too if you wish.

**Reply by afpineda** [reply]
> Hi, I have a few questions and would like to join the Discord server but the link is outdated. Could you update it?

Try this link (available for 7 days):

[https://discord.gg/uV6YTAVF](https://discord.gg/uV6YTAVF)
